.huskyrc

```diff
@@ -0,0 +1,5 @@
+{
+  "hooks": {
+    "pre-commit": "lint-staged"
+  }
+}
```

Comment:
Ideally should have a blank line here to keep GitHub (and linux) happy but not a blocker